File: IsPrime/Prime.c
#include<stdio.h>
#include <stdbool.h>



int main(){
	
	IsPrime(10);
	
	
	return 0;
}


int IsPrime(int number){
	
	bool x = true;
	int i;
    for (i = 2; i < number; i++)
    {
        if (number%i==0)
        {
            x = false;
        } 

    }
    

    if (x == true)
    {
        printf("this number is prime %d", number);
    }
    
    else
    {
    	printf("this number is not prime %d", number);
    }
	
}

File: IsPrime/Prime.py



def IsPrime(number):
        
    x = True
    i=2

    for  i in range(i,number):
            
        if (number%i==0):
                
            x = False


    if (x == True):
            
        print(f"this number is prime -----> {number}")
            
    else:
    
        print(f"this number is not prime ----> {number}")
            
                 

IsPrime(10)            

            
        